submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,65 @@
+const allCards = document.querySelectorAll(".flipper");
+const arrOfCards = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+let unopenedCards = arrOfCards.length;
+let firstCard;
+let secondCard;
+let clikCounter = 0;
+
+const checkCard = function (event) {
+    const checkMatch = function () {
+        if (clikCounter == 0) {
```

Comment:
Please use `===` instead of `==` everywhere.

Reply:
== is faster than ===, isn't it?

Reply:
`===` is actually faster, because if types are different it just returns `false`. If types are different and the operator is `==`, than JavaScript has to do different type operations to be able to compare two values.

Reply:
Fixed it.